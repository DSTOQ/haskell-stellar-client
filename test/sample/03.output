TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            PublicKeyEd25519
              { _publicKeyEd25519 =
                  34969225231759973836359833576240397618815352045076874891248687966810494758086
              }
        , _fee = Fee { _fee = 100 }
        , _seqNum = SequenceNumber { _sequenceNumber = 40695085712540286 }
        , _timeBounds = Nothing
        , _memo = MemoNone
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    Payment
                      PaymentOp
                        { _destination =
                            PublicKeyEd25519
                              { _publicKeyEd25519 =
                                  97678337911768558865512749678068891183310559961497594465697983260983599974155
                              }
                        , _asset =
                            AssetCreditAlphanum4
                              AssetCode4 { _assetCode4 = 1213483854 }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    48719781938602096453436114010251764830087310142640098156249894279754447416792
                                }
                        , _amount = 50000000
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint "168c6cc6"
          , _signature =
              Signature
                "f2b23d6b1972240e590258e9d052434633dddfa88bf85fdf0b0eeded32776bd4140b7b19f336e817a08baf461e038fe649cfb01d5b65cc838293021bfc7ff00f"
          }
      ]
  }