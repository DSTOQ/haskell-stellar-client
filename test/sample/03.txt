AAAAAE1P4U0k6OQcBBEz7AOm+zk2Bxyu4HEvD8osch0WjGzGAAAAZACQk/cAAAJ+AAAAAAAAAAAAAAABAAAAAAAAAAEAAAAA1/P65bZqgw6O/3s1hEygpT/j/DgqwGUM0hMt4greRwsAAAABSFRLTgAAAABrtmtcRKQP2CxErmtutezBy3jNwGphtvgznsq7cW1t2AAAAAAC+vCAAAAAAAAAAAEWjGzGAAAAQPKyPWsZciQOWQJY6dBSQ0Yz3d+oi/hf3wsO7e0yd2vUFAt7GfM26Begi69GHgOP5knPsB1bZcyDgpMCG/x/8A8=
