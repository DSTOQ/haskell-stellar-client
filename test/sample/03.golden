TransactionEnvelope {transaction = Transaction {sourceAccount = PublicKeyEd25519 {unPublicKeyEd25519 = 34969225231759973836359833576240397618815352045076874891248687966810494758086}, fee = Fee {unFee = 100}, seqNum = SequenceNumber {unSequenceNumber = 40695085712540286}, timeBounds = Nothing, memo = MemoNone, operations = VarLen {unVarLen = [Operation {sourceAccount = Nothing, body = Payment (PaymentOp {destination = PublicKeyEd25519 {unPublicKeyEd25519 = 97678337911768558865512749678068891183310559961497594465697983260983599974155}, asset = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1213483854}) (PublicKeyEd25519 {unPublicKeyEd25519 = 48719781938602096453436114010251764830087310142640098156249894279754447416792}), amount = 50000000})}]}}, signatures = VarLen {unVarLen = [DecoratedSignature {hint = SignatureHint {unSignatureHint = 378301638}, signature = Signature {unSignature = VarLen {unVarLen = "\242\178=k\EMr$\SOY\STXX\233\208RCF3\221\223\168\139\248_\223\v\SO\237\237\&2wk\212\DC4\v{\EM\243\&6\232\ETB\160\139\175F\RS\ETX\143\230I\207\176\GS[e\204\131\130\147\STX\ESC\252\DEL\240\SI"}}}]}}